hostinfo/lsyncd.lua

```diff
@@ -0,0 +1,142 @@
+--[[
+Copyright 2015 Rackspace
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--]]
+local HostInfo = require('./base').HostInfo
+local Transform = require('stream').Transform
+local misc = require('./misc')
+local run = misc.run
+local safeMerge = misc.safeMerge
+local getInfoByVendor = misc.getInfoByVendor
+local async = require('async')
+--------------------------------------------------------------------------------------------------------------------
```

Comment:
Sort of, Tim's mentioned a few times that this is a more optimal coding style since luajit caches the whole library otherwise so its best to declare which bits we need at the start